common/internal-types/src/account.rs

```diff
@@ -11,64 +11,33 @@ pub enum AccountType {
     /// Node is not announced.
     NotAnnounced,
     /// Node is announced with a multi-address
-    Announced { multiaddr: Multiaddr, updated_block: u32 },
+    Announced(Multiaddr),
 }
 
 impl Display for AccountType {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         match &self {
             Self::NotAnnounced => {
-                write!(f, "Not announced")
+                write!(f, "not announced")
             }
-            Self::Announced {
-                multiaddr,
-                updated_block,
-            } => f
-                .debug_struct("AccountType")
-                .field("MultiAddr", multiaddr)
-                .field("UpdatedAt", updated_block)
-                .finish(),
+            Self::Announced(multiaddr) => write!(f, "announced as {multiaddr}"),
```

Comment:
The DHT will allow more than 1 multiaddress per announcement, should we not make it a vec in this data object as well?

Reply:
Sure, I can update that.